tests/test_wave_01.py

```diff
@@ -60,13 +60,16 @@ def test_get_task_not_found(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
+    assert "details" in response_body
+    assert response_body == {
+        "details": f"Task 1 not found"
```

Comment:
Good job with this assertion! No need to use an f string here since there's no variable interpolation.